solr/core/src/java/org/apache/solr/cloud/CloudUtil.java

```diff
@@ -125,7 +125,7 @@ public static boolean replicaExists(ClusterState clusterState, String collection
   public static String unifiedResourcePath(SolrResourceLoader loader) {
     return (loader instanceof ZkSolrResourceLoader) ?
             ((ZkSolrResourceLoader) loader).getConfigSetZkPath() + "/" :
-            loader.getConfigDir() + File.separator;
+            loader.getConfigPath().toString() + File.separator;
```

Comment:
Referencing File.separator seems wrong here; just use '/'.  toString() is automatic as well.